bucket_map/src/bucket_storage.rs

```diff
@@ -564,16 +564,6 @@ mod test {
         let ix = 0;
         assert!(storage.is_free(ix));
         assert!(storage.occupy(ix, false).is_ok());
-        assert!(storage.occupy(ix, false).is_err());
```

Comment:
occupying the index no longer marks the entry as in use for the bit-based version of `occupy`. So, the rest of this test is not valid.